tests/test_tree.py

```diff
@@ -144,7 +164,7 @@ def test_new_columns_constructor():
         CHAID.OrdinalColumn(age, name="age", metadata=metadata),
     ]
     tree = CHAID.Tree(cols, CHAID.NominalColumn(income), {'min_child_node_size': 1})
-    assert tree.tree_store[0].split.groupings == "[['0-5'], ['6-10'], ['11-15']]"
+    assert tree.tree_store[0].split.groupings == "[['0-5'], ['6-10', '11-15']]"
```

Comment:
One again, can we properly justify this

Reply:
It's basically the difference between chi-squaring all them of them individually. As in a 3x3 chi matrix, which returns sig, or comparing each split against eachother and these two: `['6-10', '11-15']` when in a 2x2 matrix get p value > alpha merge. It's right on the level and is something that happens when your base size is like 10 and your . Rather than writing code for that specific use case, I'm happy with it merging them. The base size is 20 and won't happen if you have actual sufficient sample sizes.